leshan-server-demo/pom.xml

```diff
@@ -26,7 +26,21 @@ Contributors:
     <name>leshan - server demo</name>
     <description>A LWM2M demonstration server running an embedded Jetty server</description>
 
+    <repositories>
+        <repository>
+            <id>jitpack.io</id>
+            <url>https://jitpack.io</url>
+        </repository>
+    </repositories>
+
     <dependencies>
+
+        <dependency>
+            <groupId>com.github.jmdns</groupId>
+            <artifactId>jmdns</artifactId>
+            <version>jmdns-3.5.1</version>
```

Comment:
Why not using the maven [central one](https://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.jmdns%22) ?